src/jsx.d.ts

```diff
@@ -88,2621 +41,468 @@ export namespace JSXInternal {
 		children: any;
 	}
 
-	export type DOMCSSProperties = {
-		[key in keyof Omit<
-			CSSStyleDeclaration,
-			| 'item'
-			| 'setProperty'
-			| 'removeProperty'
-			| 'getPropertyValue'
-			| 'getPropertyPriority'
-		>]?: string | number | null | undefined;
-	};
-	export type AllCSSProperties = {
-		[key: string]: string | number | null | undefined;
-	};
-	export interface CSSProperties extends AllCSSProperties, DOMCSSProperties {
-		cssText?: string | null;
-	}
+	export import DOMCSSProperties = preact.DOMCSSProperties;
```

Comment:
Whilst I'd like to add `@deprecated` jsdoc comments here to steer folks to the Preact namespace, turns out you can't: `export import` forwards on the comments of the type, it itself cannot be directly annotated.

Reply:
Update: This is a bug in TS, `export import` specifically is missing this functionality at the moment